packages/cli/commands/project/create.js

```diff
@@ -32,42 +31,93 @@ const loadAndValidateOptions = async options => {
   }
 };
 
-exports.command = 'create [path]';
+exports.command = 'create';
 exports.describe = false;
 
 exports.handler = async options => {
   loadAndValidateOptions(options);
 
-  const { path: projectPath, name } = options;
   const accountId = getAccountId(options);
 
-  trackCommandUsage('project-create', { projectPath }, accountId);
-
-  const cwd = projectPath ? path.resolve(getCwd(), projectPath) : getCwd();
+  const { name, template, location } = await prompt([
+    {
+      name: 'name',
+      message: '[--name] Give your project a name:',
+      when: !options.name,
+      validate: input => {
+        if (!input) {
+          return 'A project name is required';
+        }
+        return true;
+      },
+    },
+    {
+      name: 'location',
+      message: '[--location] Where should the project be created?',
+      when: !options.location,
+      default: answers => {
+        return path.resolve(getCwd(), answers.name || options.name);
+      },
+      validate: input => {
+        if (!input) {
+          return 'A project location is required';
+        }
+        return true;
+      },
+    },
+    {
+      name: 'template',
+      message: () => {
+        return options.template &&
+          !PROJECT_TEMPLATES.find(t => t.name === options.template)
+          ? `[--template] Could not find template ${options.template}. Please choose an available template.`
```

Comment:
Should we output a list of available templates here?

Reply:
Yep, this `rawlist` is actually shown with all available templates. Choices are added here: https://github.com/HubSpot/hubspot-cli/pull/588/files#diff-846ab0ff6b94aadce7bb7339adf043c6f81967013a7dfba6dd5e5ee83a055985R85